onboarding_lite_labs.prolific

```diff
@@ -0,0 +1,108 @@
+Lab: Bug with " / " in org name
+
+### Bug description
+
+Bug with " / " in org name (also possibly space name) makes appmetrics dashboard not work
+
+### Slack thread
+
+https://vmware.slack.com/archives/CH1RH9KH7/p1605734027011800
```

Comment:
Since we're thinking of potentially opening up the CF Onboarding Week backlog to the broader OSS community, perhaps we should refrain from using vmware slack references and Jira tickets. These labs look really great for getting some hands on experience with fixing bugs in a CF env! They do seem to be app metrics-specific so perhaps this can be renamed to be "app_metrics_labs" or "onboarding_lite_app_metrics_labs"?

Reply:
Totally makes sense; yeah these labs are what I had the most concern around